test/wizard/wizard.spec.js

```diff
@@ -92,18 +97,17 @@ describe('Component:  pfWizard', function () {
     initializeWizard();
   };
 
-  beforeEach(function () {
+  var setupWizard = function(wizardHtml) {
     setupWizardScope();
-    var modalHtml = $templateCache.get('test/wizard/wizard-container.html');
+
+    var modalHtml = $templateCache.get(wizardHtml);
     element = compileHtml(modalHtml, $scope);
     $scope.$digest();
-
-    // there are two dependent timeouts in the wizard that need to be flushed
```

Comment:
The comment used here relates a nice bit of info about the need for running ```concurrent $timeout.flush()``` Should we look at keeping at least one of them?